hammett/stopwatch/collector.py

```diff
@@ -0,0 +1,84 @@
+"""The module contains a decorator that wraps handlers to track time."""
+
+import asyncio
+import functools
+import logging
+import time
+from typing import TYPE_CHECKING
+
+from hammett.conf import settings
+from hammett.core.exceptions import ImproperlyConfigured
+from hammett.stopwatch.event_loop import StopWatchSelector
+from hammett.utils.handler import wraps_handler
+from hammett.utils.module_loading import import_string
+
+if TYPE_CHECKING:
+    from typing import Any
+
+    from hammett.core.constants import HandlerStats
+    from hammett.stopwatch.stats import BaseStatsProcessor
+    from hammett.types import Handler
+
+LOGGER = logging.getLogger(__name__)
+
+
+@functools.cache
+def get_stopwatch_stats_processor() -> 'type[BaseStatsProcessor]':
+    """Import a stopwatch stats processor by the `STOPWATCH_STATS_PROCESSOR` of the module path
+    and returns it.
```

Comment:
`Import a stopwatch stats processor from the path specified in the `STOPWATCH_STATS_PROCESSOR` setting and return it.`